uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -149,7 +149,8 @@
 <xsl:variable name="ABSOLUTE_MEDIA_PATH" select="concat($CONTEXT_PATH,'/',$MEDIA_PATH)"/>
 <xsl:variable name="SKIN_RESOURCES_PATH" select="concat('/',$MEDIA_PATH,'/',$SKIN,'/skin.xml')"/>
 <xsl:variable name="SKIN_PATH" select="concat($ABSOLUTE_MEDIA_PATH,'/',$SKIN)"/>
-<xsl:variable name="PORTAL_SHORTCUT_ICON" select="concat($CONTEXT_PATH,'/favicon.ico')" />
+<xsl:variable name="SHORTCUT_ICON" select="concat($CONTEXT_PATH,'/favicon.ico')" />
+<xsl:param name="PORTAL_SHORTCUT_ICON"><xsl:value-of select="$SHORTCUT_ICON"/></xsl:param>
```

Comment:
To avoid needing sometthing like `@Value("${org.apereo.portal.layout.faviconPath:${portal.context}/favicon.ico}")` (which won't work, I think), how about...

```
<xsl:variable name="SHORTCUT_ICON">/favicon.ico</xsl:variable>
<xsl:param name="PORTAL_SHORTCUT_ICON" select="concat($CONTEXT_PATH, $SHORTCUT_ICON)"/>
```

Reply:
There isn't more needs to modify anything.